tests/testthat/test-osmodel.R

```diff
@@ -0,0 +1,29 @@
+
+test_that("parametrize filled all the gaps", {
+    os_mod <- LogLogisticOs()
+    h_link <- HazardLink(
+        stan = StanModule(
+            functions = "bla",
+            data = "bla",
+            parameters = "bla",
+            transformed_parameters = "bla",
+            generated_quantities = "bla",
+            priors = list(),
+            inits = list()
+        ),
+        parameters = "bla",
+        contribution = "bla"
+    )
+    os_mod_f <- parametrize(os_mod, h_link)
+
+    all(sapply(
+        c("link_arguments", "link_log_hazard_contribution", "link_arguments_as_par"),
+        grepl, os_mod_f@stan@functions
+    )) != TRUE
```

Comment:
We have used the testthat package here which means we need to use expectation functions i.e.  

```
expect_true(  1 ==1 )
```

Would recommend taking a look at: https://www.r-bloggers.com/2019/11/automated-testing-with-testthat-in-practice/

Am happy to have a chat to go over this if it would help

Reply:
I hope my last update fixed this issue.